src/pages/CatInfoPage.js

```diff
@@ -38,7 +39,8 @@ export default function CatInfoPage() {
           type="button"
           onClick={() => navigate('/ProfilePage', { replace: true })}
```

Comment:
I think the path should always be in lower case (e.g. /profilepage or /profile-page)

Reply:
yes, thanks for pointing this out - i have put it in the next userstory with quite a lot code clean up